source/standards/dns-hosting.html.md.erb

```diff
@@ -1,24 +1,23 @@
 ---
-title: DNS hosting
+title: How to manage DNS
 expires: 2018-01-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-To make your services available over the internet they will need DNS records to let people know where they are.
-`service.gov.uk` domains are delegated to [nameservers](https://www.gov.uk/service-manual/technology/get-a-domain-name#check-where-youll-host-your-dns) controlled by service teams.
+You need accurate DNS records to ensure your service is available to your users. For example, services on GOV.UK domains are delegated to [nameservers](https://www.gov.uk/service-manual/technology/get-a-domain-name#check-where-youll-host-your-dns) controlled by GDS service teams.
 
+As part of your DNS strategy and planning we recommend your:
 
-## Principles
+* DNS entry configuration comes from a version-controlled repository such as GitHub, unless autogenerated
+* overall service strategy includes nameserver availability
 
-- DNS entry configuration should come from a version-controlled repository unless autogenerated
-- Consider nameserver availability as part of your overall service availability
+When you implement your DNS strategy we recommend using:
 
+* [Amazon Route 53](https://aws.amazon.com/route53) as a cloud DNS web service
+* [Google Cloud DNS](https://cloud.google.com/dns/) or [Dyn](https://dyn.com) if you want greater nameserver availability
```

Comment:
Are we still using Dyn? Can you check these options with @stephenharker?

Reply:
We are finishing off a migration away from Dyn towards AWS/Google but in principle, they are a valid supplier.

Reply:
thanks for looking at that PR and your comment, let's leave the Dyn note in the guidance for now.